packages/axios-extension/src/abap/adt-catalog/handlers/transport.ts

```diff
@@ -102,6 +102,7 @@ function getLockedTransport(doc: Document): TransportRequest[] {
 
 /**
  * Convert transport request in XML element of ADT response to typed object.
+ *
```

Comment:
Of course it is only a comment change (most likely done automatically with `lint --fix`) but still I would prefer not to have it in this PR, so that all your changes are limited to one module.

Reply:
Not relevant anymore, file was deleted, done by merge from main